extension/httpforwarder/README.md

```diff
@@ -0,0 +1,27 @@
+# HTTP Forwarder Extension
+
+This extension accepts HTTP requests, optionally adds headers to them and forwards them.
+The RequestURIs of the original requests are preserved by the extension.
+
+## Configuration
+
+* `incoming`: HTTP config settings for HTTP server listening to requests.
+  * `endpoint` (default: `localhost:6060`): The host to which requests should be forwarded to.
+* `outgoing`: HTTP config settings to use for forwarding requests.
+  * `endpoint` (**required**): The target to which requests should be forwarded to.
+  * `headers` (default: `nil`): Additional headers to be added to all requests passing through the extension.
+  * `timeout` (default: `10s`): How long to wait for each request to complete.
+
+### Example
+
+```yaml
+  http_forwarder:
+    incoming:
```

Comment:
Not blocking, do you think actually ingress and egress make more sense?

Reply:
yes, it probably makes more sense. Renamed them.